README.md

```diff
@@ -75,3 +75,11 @@ We have a few samples to demonstrate this plugin:
 - [React native](./samples/react-native-app/)
 
 Much of the logic for adding the attributes originated in the [transform-react-qa-classes](https://github.com/davesnx/babel-plugin-transform-react-qa-classes/) plugin.
+
+## Getting Help
+
+Please refer to our [Knowledge Base article](https://help.fullstory.com/hc/en-us/articles/360049493054-FullStory-s-Annotate-React-plugin-for-Web-Native) or  contact support@fullstory.com for additional help.
```

Comment:
Ye4ah, this should be `mobile-support@fullstory.com` and also there's a double space at `or  contact`